nodejs/json-rpc-relay/integration.tests/contracts/mainnet/write.contract.mainnet.test.js

```diff
@@ -0,0 +1,40 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+const { ethers } = require("ethers");
+const urlHandler = require('../../../../handlers/url.handler');
+const fs = require('fs');
+const path = require('path');
+const privateECDSAAccount = process.env.MAINNET_OPERATOR_PRIVATE_KEY;
+
+const mainnetContractConfig = {
+    id: ``,
+    sol_id: ``
+}
+
+describe('[Ethers] Signed RawTransaction', () => {
+
+    test('Mainnet | Should make raw transaction into Contract', async () => {
+        // Arrange
+        const provider = new ethers.providers.JsonRpcProvider(urlHandler.getJsonRpcMainnet());
+        const signer = new ethers.Wallet(privateECDSAAccount, provider);
+        const contractPath = path.join(__dirname, '/');
+
+        const contractJson = fs.readFileSync(`${contractPath}/../artifact/example.contract.json`);
+        const ABI = JSON.parse(contractJson);
+        const sampleContract = new ethers.Contract(mainnetContractConfig.sol_id, ABI, signer)
+        const newCreator = `ARKHIA ${new Date().toDateString()}`;
+
+        // Act
+        const changeCreator = await sampleContract.setCreator(newCreator);
+        await changeCreator.wait();
```

Comment:
Test fails first round if we dont wait to be written

Reply:
Contract call works very well.

Tested and trust 
<img width="1083" alt="image" src="https://github.com/BCWResearch/arkhia-playground/assets/113974710/11ab2810-9871-44dc-bedd-b2ee1d36af4c">